### DISCUSSION TITLE
Inquiry on Documentation Materials

### DISCUSSION THREAD

**Discussion by DotJARgon** [DualSPHysics/DualSPHysics#205] [DUALSPHYSICS_V5.2]
Hello everyone! I was wondering if anyone has made any UML diagrams, or any diagrams of the memory layout of the DualSPhysics application! My team is working on dynamically breaking solids in dualsphysics, and I am working on finding what all I need to modify for the floatings to make sure I do not break the application! If you know of any examples of breaking up floating solids that already exists please let me know! Thank you very much! I will share the results as well once my team has dynamic breaking of floating solids.

**Comment by alexbexe** [comment]
Dear user

In the SPHERIC community you can find some groups implementing codes focused on fluid dynamics and different ones working on solid mechanics... However, as far as we know, there is SPH code where the 2 physics are implemented in the same framework...
In the case of DualSPHysics, we include a coupling with CHRONO for solid collisions only .

You can find more information about solid mechanics within SPH at https://www.spheric-sph.org/

Regards
